docs/registry/create/_index.md

```diff
@@ -68,73 +70,72 @@ To ensure the client interface you create returns the expected results, use the
-   Import your custom model and API into the main program and register them with your chosen SDK.
-   This main program is the "entry point" to your module.
+If you are using unique hardware that does not already have an [appropriate API](/program/apis/#component-apis) defined to support it, you can use the [generic API](/components/generic/) to add support for that unique hardware type to your smart machine.
 
-3. [Compile or package](#compile-the-module-into-an-executable) the module into a single executable that can receive a socket argument from Viam, open the socket, and start the module at the entry point.
+Some use cases may require you to define a new API, or to deploy custom components using a server on a remote part.
+For more information, see [Advanced Modular Resources](/registry/advanced/).
 
-{{% /tab %}}
-{{< /tabs >}}
+## Create a custom module
+
+A custom module wraps one or more [models](/registry/key-concepts/#models).
+To create a custom module, follow these steps:
+
+1. [Code a new resource model](#code-a-new-resource-model) server.
+2. [Code a main program](#code-a-main-entry-point-program).
+3. [Compile or package](#compile-the-module-into-an-executable) the module into a single executable.
 
 ### Code a new resource model
 
-The following example module registers a modular resource implementing Viam's built-in [Base API](/components/base/#api) [(rdk:service:base)](/modular-resources/key-concepts/#models) as a new model, `"mybase"`, using the model family `acme:demo:mybase`.
-For more information, see [Naming your model](/modular-resources/key-concepts/#naming-your-model-namespacerepo-namename).
+{{% alert title="Naming your model" color="tip" %}}
 
-{{% alert title="Tip" color="tip" %}}
+Use the naming schema: `namespace:repo-name:name`.
 
-Name your model with all lowercase letters for optimal performance with Viam's SDKs.
-For example, `mybase` or `my-cool-sensor`.
+For more information see [Naming your model](/registry/upload/#naming-your-model-namespacerepo-namename).
 
 {{% /alert %}}
 
-{{< tabs name="Sample SDK Code">}}
-{{% tab name="Python"%}}
+{{< tabs >}}
+{{% tab name="Python" %}}
+Your new resource model server must have all the methods that the Viam RDK requires, and should match the built-in API client {{< glossary_tooltip term_id="subtype" text="subtype" >}} like ['rdk:component:base'](https://python.viam.dev/autoapi/viam/components/base/index.html).
+
+Create a folder for your module and save your code as a file named <file>my_modular_resource.py</file> inside.
+
+The following example module registers a modular resource implementing Viam's built-in [Base API](/components/base/#api) [(rdk:service:base)](/registry/key-concepts/#models) as a new model, `"mybase"`, using the model family `acme:demo:mybase`.
 
 <file>my_base.py</file> implements "mybase", a custom model of the base component.
```

Comment:
Opt: could remove this line now that we have line 133 above, which does it better.